environment.yml

```diff
@@ -6,6 +6,7 @@ dependencies:
   - ipython
   - matplotlib
   - pdal
+  - opencv
```

Comment:
This makes me think. The list of dependencies is starting to grow a bit. Sometimes it can be difficult to resolve conflicts (I've had a lot of issues of conflicts between gdal and opencv, the latter only compatible with old versions of gdal).
Maybe we should make some of these dependencies soft and add a test at the import. If opencv is not available, set opencv_avail=False. Then in any of the function where needed, if this is False, simply raise an error.

Reply:
@adehecq  I have done exactly that. But as I mentioned, this is in the wrong PR. Sorry!!

Reply:
Never mind, I just saw it's what you did! Then opencv should be listed as soft dependency (maybe same for pdal?).